actions/cloud.tsx

```diff
@@ -121,11 +125,14 @@ export function validateWorkspaceBusinessEmail() {
     trackEvent('api', 'api_validate_workspace_business_email');
     return async () => {
         try {
-            await Client4.validateWorkspaceBusinessEmail();
+            const res = await Client4.validateWorkspaceBusinessEmail();
+            if (res.data.is_valid === 'true') {
```

Comment:
```suggestion
            if (res.data.is_valid.toLowerCase() === 'true') {
```

Reply:
if this were a boolean, then this could become

```ts
        try {
            return await Client4.validateWorkspaceBusinessEmail().data.is_valid;
```